packages/core/src/api/guild.ts

```diff
@@ -1318,26 +1282,6 @@ export class GuildsAPI {
 		return this.rest.get(Routes.template(templateCode), { auth, signal }) as Promise<RESTGetAPITemplateResult>;
 	}
 
-	/**
-	 * Creates a new template
-	 *
-	 * @see {@link https://discord.com/developers/docs/resources/guild-template#create-guild-template}
-	 * @param templateCode - The code of the template
-	 * @param body - The data for creating the template
-	 * @param options - The options for creating the template
-	 */
-	public async createTemplate(
```

Comment:
This removal will conflict with the change in #11030

Reply:
Not a problem.